### PR TITLE
some interactive scenarios and blogs added

### PR DESCRIPTION
**Pull request by kranurag7**
- updated logo and exam brief 
- Added interactive labs by Sysdig and Promlabs 
- Some other helpful blogs

### CODE REVIEW

**Comment by walidshaari** [comment]
Thank you @kranurag7 much appreciated.

**Comment by mehabhalodiya** [comment]
LGTM! Thanks for contributing @kranurag7 
Little suggestion..

File: README.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # Linux Foundation Prometheus Certified Associate PCA Beta
 
+<p align="center">
+  <img width="240" src="https://training.linuxfoundation.org/wp-content/uploads/2022/04/Training_Badge_Prometheus_V2-2-300x300.png">
+</p>
+
 Prometheus Certified Associate (PCA) Certification
+### Exam Brief 
+- Duration - 90 minutes 
+- Cost - $250 One (1) year exam eligibility, with a free retake within the year.
```
Comment (mehabhalodiya) [inline README.md]:
```suggestion
- Cost - $250, with a free retake within the year. This registration is valid for 12 months from the date of purchase, or when your corporate subscription expires (whichever happens first).
```

**Review by mehabhalodiya — CHANGES_REQUESTED** [review]
Minor changes can be done 🙂

File: README.md (hunk 1 above)
Comment (kranurag7) [reply README.md]: /LGTM, Thanks Meha!

File: README.md [hunk 2]
```diff
@@ -47,4 +57,7 @@ Prometheus Certified Associate Beta exam
 1. [Linux Foundation official course LFS241 monitoring systems and services with prometheus ](https://training.linuxfoundation.org/training/monitoring-systems-and-services-with-prometheus-lfs241/)
 1. [PluralSight/acloud guru Prometheus deep dive](https://acloudguru.com/course/prometheus-deep-dive)
 
-
+### Interactive Scenarios 
+- [Sysdig Hands on Interactive Scenarios](https://learn.sysdig.com/page/open-source)
+- [promlabs](https://demo.promlabs.com)
+- [promlens](https://demo.promlens.com)
```
Comment (mehabhalodiya) [inline README.md:63]: Repeatedly written; remove last line content and keep a blank line at last.

**Review by mehabhalodiya — CHANGES_REQUESTED** [review]
Thanks for updating!

**Pull request merged**